app/views/welcome/index.html.erb

```diff
@@ -1 +1,19 @@
-<h1>Welcome</h1>
+<!-- This needs to be centered and very large font -->
+<h1>Welcome to Viewing Party</h1>
+
+<!-- This needs to be centered and significantly smaller font -->
+<h5>This application lets you explore movies and create a viewing party event for you and your friends to watch a movie together.</h5>
+
+<!-- This needs to be centered under the description -->
+<%= form_tag "/dashboard" do %>
+  <%= label_tag :email %>
+  <%= email_field_tag :email %>
+
+  <%= label_tag :password %>
+  <%= password_field_tag :password %>
+
+  <%= submit_tag 'Sign In', class: :button %>
```

Comment:
Nice form!  And taking advantage of the automated method to post from the `submit_tag`

Reply:
Thanks!